doc/integrator/api.rst

```diff
@@ -34,3 +34,10 @@ In the file ``geoportal/demo_geoportal/static-ngeo/api/index.js`` you can custom
    config.queryableLayers = [<layer a>, <layer b>, ...];
 
 The API help is in the folder ``geoportal/<package>_geoportal/static/apihelp/``.
+
+In order for a layer to be accessible via API, the layer must be in the themes::
+
+ * Add the layer
+ * Select the api interface
+ * Add the layer to a group (the group should already be visible by the api,
```

Comment:
Suggestion:

* Add the layer to a group which is visible in the API.

Note: a group is visible in the API if it is in a theme whose API configuration option is activated.

Reply:
Adapted and updated, what do you think?

Reply:
I like this except that the "the" should be removed in 
"whose the 'api' interface"

Reply:
OK, updated :-), thanks